packages/main/src/controllers/message/MessageService.ts

```diff
@@ -235,6 +238,37 @@ export class MessageService {
             )
           }
         }
+      } else if (messageType === CredentialRevocationMessage.type) {
+        const msg = JsonTransformer.fromJSON(message, CredentialRevocationMessage)
+
+        const credentials = await agent.credentials.findAllByQuery({ threadId: msg.threadId })
+        if (credentials && credentials.length > 0) {
+          for (const credential of credentials) {
+            const isRevoke = Boolean(
+              credential.getTag('anonCredsRevocationRegistryId') &&
+                credential.getTag('anonCredsCredentialRevocationId'),
+            )
+            isRevoke &&
+              (await agent.modules.anoncreds.updateRevocationStatusList({
```

Comment:
Here it would be nice to check status to see if an error happened while revoking the credential. Otherwise, there is a possibility of sending a notification when the credential is not actually revoked (updateRevocationStatusList does not throw errors in normal operation).